cfnet/datasets.py

```diff
@@ -165,14 +159,39 @@ def __init__(self, data_configs: Dict):
         # prepare data
         self.prepare_data()
 
+    def name_to_config(self, data_name):
+        with open('../assets/configs/{}.json'.format(data_name)) as json_file:
+            data = json.load(json_file)
+            data_configs['data_name']=data_name
+            data_configs['discret_cols']=data['discret_cols']
+            data_configs['continous_cols']=data['continous_cols']
+            data_configs['imutable_cols']=data.get('imutable_cols', [])
+            data_configs['sample_frac']=data.get('sample_frac', [])
+            data_configs['normalizer']=data.get('normalizer', [])
+            data_configs['encoder']=data.get('encoder', [])
+            data_configs['data_dir'] = self.download_data(data_name)
+        return data_configs
+
+    def download_data(self,data_name):
+        url='https://github.com/BirkhoffG/cfnet/raw/master/assets/data/{}.csv'.format(data_name)
+        path=os.getcwd()
+        print(path)
+        if not os.path.exists("cf_data"):
+            os.makedirs("cf_data")
+        path = os.path.join(path, "cf_data", "{}.csv".format(data_name))
```

Comment:
As you are using the `Path` library, consider to write it as `path /  'cf_data'  / f'{data_name}.csv'`.